Source/ASDisplayNode.mm

```diff
@@ -3179,7 +3211,10 @@ - (ASEventLog *)eventLog
   if (self.debugName.length > 0) {
     [result addObject:@{ @"debugName" : ASStringWithQuotesIfMultiword(self.debugName)}];
   }
-  
+  if (self.accessibilityIdentifier.length > 0) {
+    [result addObject:@{ @"axId": ASStringWithQuotesIfMultiword(self.accessibilityIdentifier) }];
```

Comment:
Since this is the `-debugDescription` we can afford to be more verbose. The difference in output string is:

`<MYNode: 0x00000001; axId = "Like button"> vs. <MYNode: 0x00000001; "Like button">`

I'll find some way to expose this behavior more clearly in headers/comments.